samples/security/spm/spm_nspe/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _spm_psoc6:
+
+SPM base NS environment example for PSoC6  CM4
+###########
+
+Overview
+********
+A simple SPM base NS environment example for PSoC6  CM4.
+Needs Secure part on CM0+
+
+CM4(spm_nspe) sends debug log to Uart Port5 which is available through USB connection with the PSoC6 kit.
+CM0+(spm_spe) send debug messages to UART port 12(pin0(RX)/pin1(TX)) � connect external UART to UASB convertor to these pins on kit 
+
+Project description:
+- CM0p starts CM4 core
+- CM4 writes the message with the counter in loop to the shared memory block, triggers interrupt to CM0p core and wait on response.
+- Once interrupt received, CM0p prints out received message and initiate response interrupt to CM4.
+
+Building and Running
+********************
+cmake -G"Eclipse CDT4 - Ninja" -DBOARD=cy8ckit_062_wifi_bt_m0 ..
+dowload zephyr.elf by openOCD with configuration:
+GDB port: 3333
+Telnet port: 4444
+Config options:
+--search "${cy_tools_path:openocd}/scripts"
+-c "gdb_memory_map enable"
+-c "gdb_flash_program enable"
+-c "interface cmsis-dap"
+-c "transport select swd"
+-c "source [find interface/cmsis-dap.cfg]"
+-c "source [find target/psoc6.cfg]"
+
+Sample Output
+=============
+
+***** Booting Zephyr OS v1.12.0-rc2-48-g2c3c674aa *****
```

Comment:
Console output should also use a ``.. code-block`` directive:

    .. code-block: console

       ***** Booting Zephyr OS v1.12.0-rc2-48-g2c3c674aa *****
       CM4: Main